data/Templates/FhirToHL7v2/Segments/_PID.liquid

```diff
@@ -17,7 +17,13 @@
         {%- assign birthDate = birthInfo.valueDateTime -%}
     {%- endif -%}
 {%- endif -%}
-{%- assign patientAddress = patient.address | slice_array: 0, 1 -%}
+{%- assign patientMother = patientRelatedPersons | select: 'resource.relationship[].coding[].code', 'MTH' | get_index: 0 -%}
+{%- assign patientMotherIdentifierCode = patientMother.resource.identifier | select: 'type.coding[].system', 'http://terminology.hl7.org/CodeSystem/v2-0203' | get_index: 0 -%}
+{%- assign patientMotherIdentifierCodeRef = patientMotherIdentifierCode.assigner.reference | slice: 13, -1 -%}
+{%- assign patientMotherOrg = patientOrganizations | select: 'resource.id', patientMotherIdentifierCodeRef | get_index: 0 -%}
+{%- evaluate patientMotherCXJson using 'Mutators/PatientMotherToCX' mother: patientMotherIdentifierCode, org: patientMotherOrg -%}
+{%- assign patientMotherCX = patientMotherCXJson | from_json_string -%}
```

Comment:
Just a question as I think about how I restructured PID-3. Would it make sense to combine some of these assignments in the `Mutators/PatientMotherToCX` template? Possibly just passing in the all related persons and having the template do the filtering?

Reply:
Yep, I've been toying with the idea myself. I think the current pattern is still every so slightly messy. If you can think of a good way to clean things up then I would try it out. :)